eng/common/templates-official/job/source-build.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     pool:
       ${{ if eq(variables['System.TeamProject'], 'public') }}:
         name: $[replace(replace(eq(contains(coalesce(variables['System.PullRequest.TargetBranch'], variables['Build.SourceBranch'], 'refs/heads/main'), 'release'), 'true'), True, 'NetCore-Svc-Public' ), False, 'NetCore-Public')]
-        demands: ImageOverride -equals build.ubuntu.1804.amd64
```

Comment:
Anything under eng/common may get updated or overwritten with the next arcade DARC update.  I assume this change is being made in arcade also.